javers-core/src/main/java/org/javers/common/reflection/ReflectionUtil.java

```diff
@@ -110,6 +110,41 @@ public static Object newInstance(Class clazz, ArgumentResolver resolver){
         return result;
     }
 
+    static boolean trySetMemberRelevantToGetter(JaversMethod getter, Object onObject, Object value) {
+        if (isGetter(getter.getRawMember())) {
+            String setterName = setterNameForGetterName(getter.getRawMember().getName());
+            return trySetMemberWithSetter(setterName, getter.getGenericResolvedType().getClass(), onObject, value) ||
```

Comment:
czemu fallback na pole? spowoduje to mieszany styl Bean/Field. Jeśli ktoś ma Bean Style to powinien mieć gettery i settery, jeśli nie ma settera rzucałbym wyjątek